src/interpreter/ByteCodeGenerator.cpp

```diff
@@ -210,7 +212,7 @@ ByteCodeBlock* ByteCodeGenerator::generateByteCode(Context* context, Interpreted
         ast->generateStatementByteCode(block, &ctx);
 
 #ifdef ESCARGOT_DEBUGGER
-        if (context->debugger() && context->debugger()->enabled()) {
+        if (context->debugger() && context->debugger()->enabled() && breakpointContext.m_parsingEnabled) {
```

Comment:
What is the difference between `context->debugger()` and `context->debugger()->enabled()`?

Reply:
the `enabled()` property simplifies the enabled checks inside the debugger code, since the debugger has no access to the context. This could be changed though, if the debugger would have a pointer to to the `context->m_debugger` property. I could do it in a separate patch.

Reply:
`enable()` means that Debugger is now available, right?
Is it also means that there are some possible cases that Debugger is created but disabled during its run?
If so, I think that current version is good enough :)